jvb/src/main/kotlin/org/jitsi/videobridge/AudioSubscriptionManager.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright @ 2025 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge
+
+import org.jitsi.videobridge.message.ReceiverAudioSubscriptionMessage
+import org.jitsi.videobridge.relay.AudioSourceDesc
+import java.util.concurrent.ConcurrentHashMap
+
+class AudioSubscriptionManager(private val conference: Conference) {
+    /**
+     * A map of endpoint IDs to their audio subscriptions.
+     */
+    private val audioSubscriptions = ConcurrentHashMap<String, AudioSubscription>()
+
+    /**
+     * A map of local audio source names to a set of endpoint IDs that explicitly subscribe to the source.
```

Comment:
Define what you mean by "explicitly" (with an "Include" subscription, not All or Except)